plane2/schema/derived_schema.sql

```diff
@@ -253,7 +253,10 @@ ALTER TABLE public.controller OWNER TO postgres;
 
 CREATE TABLE public.drone (
     id integer NOT NULL,
-    draining boolean DEFAULT false NOT NULL
+    ready boolean NOT NULL,
+    draining boolean DEFAULT false NOT NULL,
+    last_heartbeat timestamp with time zone,
+    last_local_epoch_millis bigint
```

Comment:
Even though this represents a time, to the controller it is just an opaque number of milliseconds -- treating it as a clock time in the controller would be a bug.